apps/experiments/migrations/0055_consentform_conversational_consent.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 4.2.7 on 2023-11-28 10:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("experiments", "0054_consentform_capture_identifier_and_more"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="consentform",
+            name="conversational_consent",
+            field=models.CharField(
+                default="Use any of the following to accept: {yes,accept,I accept}",
```

Comment:
I'm not very happy with the approach I took, but couldn't think of a better one. Ultimately, we need a way to tell users "respond with the following word/phrase(s) to accept" in multiple languages, and only when the consent form is given through conversation. Other suggestions are super welcome

Reply:
With telegram you can create [buttons](https://core.telegram.org/api/bots/buttons). Whatsapp too but they need to go through an [approval process](https://www.freshworks.com/live-chat-software/customer-engagement/whatsapp-interactive-buttons-blog/).

Do we even know the user's language by this point? If we do we could use the LLM to produce the text with a prompt like:

> Write a short message to determine if a user agrees to continue they must type "1". Write in French.

Reply:
Also thinking that it will be simpler if we always make the user type "1" to agree.

Reply:
> Do we even know the user's language by this point? If we do we could use the LLM to produce the text with a prompt like

Depends on the message they used to initiate the conversation with, so I wouldn't really rely on that. For example, "hi" is quite language agnostic.

What we _can_ do, come to think of it, is to use the same language in which the consent form is written, since we're presenting users with the consent form in anycase :thinking:

Reply:
I think let's try to generate "Type '1' if you agree" in the language that the consent form is written in and see how that goes. If that doesn't work, we can expose a new field to allow bot creators to manually specify this sentence.